doxygen/src/pack_publish.txt

```diff
@@ -89,48 +239,116 @@ element in the PDSC file. To ensure that the pack is found by a \ref cp_KeilComP
 - Arm recommends to keep the url_A alive for at least six months so that users have time to pick up the change. Be aware
   that users who have not noticed the change might be unable to find pack updates once url_A have become unavailable.
 
-
-\section cp_PackRenaming Rename a pack
+\subsection cp_PackRenaming Renaming a pack
 
 Sometimes, it is necessary to rename a pack (because the pack vendor and/or its name have changed for example). To ensure
 that the pack is found by a \ref cp_KeilComPack "Pack Index Service" and the tools, follow this procedure:
-- Create:
-  - a new PDSC file with the updated \elem{name} and/or \elem{vendor}, for example NewVendor.NewName.pdsc. The
+- Create a new:
+  - PDSC file with the updated \elem{name} and/or \elem{vendor}, for example NewVendor.NewName.pdsc. The
     \elem{version} should start at 1.0.0.
```

Comment:
Do we mention somewhere that only release quality pack versions must be published via this index?

Reply:
Added in note on line 121.